scenes/main.tscn

```diff
@@ -1,9 +1,19 @@
-[gd_scene load_steps=5 format=4 uid="uid://7hoy2p14t6kc"]
+[gd_scene load_steps=15 format=4 uid="uid://7hoy2p14t6kc"]
 
 [ext_resource type="TileSet" uid="uid://07fq3rspk8ia" path="res://scenes/tileset.tres" id="1_0wfyh"]
 [ext_resource type="PackedScene" uid="uid://iu2q66clupc6" path="res://scenes/player/player.tscn" id="3_sugp2"]
 [ext_resource type="PackedScene" uid="uid://ipvcfv2g0oi1" path="res://scenes/npcs/talker/talker.tscn" id="4_jyhfs"]
+[ext_resource type="Resource" uid="uid://dx0uf6n0o7hv2" path="res://scenes/music_puzzle/musician.dialogue" id="5_85g3d"]
+[ext_resource type="Script" uid="uid://cfdpmffk2m4dd" path="res://scenes/music_puzzle/bard.gd" id="5_choun"]
 [ext_resource type="SpriteFrames" uid="uid://clx8sssi4tw8q" path="res://scenes/npcs/sprite_frames/sprite_frame_02.tres" id="5_tbgi4"]
+[ext_resource type="PackedScene" uid="uid://dv30a1p74xgs1" path="res://scenes/music_puzzle/musical_rock_xylophone.tscn" id="5_tefeu"]
+[ext_resource type="PackedScene" uid="uid://0ho4pvu8mn22" path="res://scenes/music_puzzle/bonfire.tscn" id="6_o6xl0"]
+[ext_resource type="Script" uid="uid://c68oh8dtr21ti" path="res://scenes/music_puzzle/music_puzzle.gd" id="8_tipki"]
+[ext_resource type="Texture2D" uid="uid://b6pfo5hc7quec" path="res://scenes/music_puzzle/clues/1.png" id="10_trceg"]
+[ext_resource type="Texture2D" uid="uid://3ieahlwwqb0t" path="res://scenes/music_puzzle/clues/2.png" id="11_a8y0u"]
+[ext_resource type="Texture2D" uid="uid://dgqq26xsu3hrf" path="res://scenes/music_puzzle/clues/3.png" id="12_jkv2x"]
+[ext_resource type="Texture2D" uid="uid://dybrvl42chyoi" path="res://scenes/music_puzzle/clues/4.png" id="13_jbj1t"]
+[ext_resource type="PackedScene" uid="uid://ckbqsrxnccwd0" path="res://scenes/music_puzzle/sheep.tscn" id="14_a8y0u"]
```

Comment:
It's kind of sad to dump all this stuff into `main.tscn` – conflicts with other branches ahoy… – but I am not sure it is worth factoring the puzzle out into yet more scenes.